eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -99,6 +99,9 @@ jobs:
       - src/coreclr/vm/*
       exclude:
       - '*'
+    - subset: coreclr_jit
+      include:
+      - src/coreclr/jit/*
```

Comment:
I presume this means all files in the `jit` directory and all subdirectories of the `jit` directory?

Reply:
looks like it.

Reply:
Yes. If you want to know more about how this works, here is the summary of how it works: https://github.com/dotnet/runtime/blob/d471a03d2c5cebfad2311a3d0408248956d0e683/eng/pipelines/evaluate-changed-paths.sh#L2-L17

And this is the logic that takes care of that:
https://github.com/dotnet/runtime/blob/d471a03d2c5cebfad2311a3d0408248956d0e683/eng/pipelines/evaluate-changed-paths.sh#L132-L183